stock_intercompany/tests/test_intercompany_picking.py

```diff
@@ -1,71 +1,84 @@
                 "name": "User Demo",
                 "email": "firstnametest@example.org",
+                "company_id": cls.company1.id,
+                "company_ids": [
+                    Command.link(cls.company1.id),
+                    Command.link(cls.company2.id),
+                ],
                 "groups_id": [
-                    (4, self.env.ref("base.group_user").id),
-                    (4, self.env.ref("stock.group_stock_user").id),
+                    Command.link(cls.env.ref("base.group_user").id),
+                    Command.link(cls.env.ref("stock.group_stock_user").id),
                 ],
             }
         )
-        company_obj = self.env["res.company"]
-        # Create 2 companies and configure intercompany picking type param on them
-        self.company1 = company_obj.create({"name": "Company A"})
-        self.company2 = company_obj.create({"name": "Company B"})
-        self.picking_type_1 = (
-            self.env["stock.picking.type"]
+        cls.picking_type_1 = (
+            cls.env["stock.picking.type"]
             .sudo()
             .search(
                 [
-                    ("company_id", "=", self.company1.id),
+                    ("company_id", "=", cls.company1.id),
                     ("name", "=", "Delivery Orders"),
                 ],
                 limit=1,
             )
         )
-        self.picking_type_2 = (
-            self.env["stock.picking.type"]
+        cls.picking_type_2 = (
+            cls.env["stock.picking.type"]
             .sudo()
             .search(
-                [("company_id", "=", self.company2.id), ("name", "=", "Receipts")],
+                [
+                    ("company_id", "=", cls.company2.id),
+                    ("name", "=", "Receipts"),
+                ],
                 limit=1,
             )
         )
 
-        self.company1.intercompany_in_type_id = self.picking_type_1.id
-        self.company2.intercompany_in_type_id = self.picking_type_2.id
+        cls.company1.intercompany_in_type_id = cls.picking_type_1.id
+        cls.company2.intercompany_in_type_id = cls.picking_type_2.id
         # assign both companies to current user
-        self.user_demo.write(
+        cls.user_demo.write(
             {
-                "company_id": self.company1.id,
-                "company_ids": [(4, self.company1.id), (4, self.company2.id)],
+                "company_id": cls.company1.id,
+                "company_ids": [
+                    Command.link(cls.company1.id),
+                    Command.link(cls.company2.id),
+                ],
             }
         )
-        # create storable product
-        self.product1 = self.env["product.product"].create(
+        cls.product1 = cls.env["product.product"].create(
             {
                 "name": "Product A",
-                "type": "product",
-                "categ_id": self.env.ref("product.product_category_all").id,
+                "type": "consu",
```

Comment:
```suggestion
                "type": "consu",
                "is_storable" : True,
```

Since we are dealing with inventory management, may we need to add is_storable boolean to True as a replacement of old "product" type value ?

https://www.odoo.com/fr_FR/forum/aide-1/new-product-product-type-odoo-18-266473